frontend/src/components/sessions.js

```diff
@@ -21,11 +25,20 @@ const DEFAULT_COLUMNS = [
  */
 export function SessionsList(props) {
     const { sessions, columns = DEFAULT_COLUMNS } = props;
+    const formattedSessions =
+        sessions.length > 0
+            ? sessions.map((session) => ({
+                ...session,
+                start_date: formatDate(session.start_date),
+                end_date: formatDate(session.end_date)
+            }))
+            : sessions;
+
     return (
         <React.Fragment>
             <h3>Sessions</h3>
             <ReactTable
-                data={sessions}
+                data={formattedSessions}
```

Comment:
I think `ReactTable` can do individual cell formatters. If you format in the cell, then the data should stay sortable.